### ISSUE TITLE
InitScript bug when teleporting

### ISSUE THREAD

**Issue by Narre** [SuperTux/supertux#698]
SuperTux version: current git version (28.04.2017)
System information: Ubuntu 17.04 64-bit

Expected behaviour: InitScript should be either stopped and triggered again when Tux is teleported, or it should be neither stopped nor triggered again.

Actual behaviour: InitScript is triggered again when Tux is teleported, but it isn't stopped when it's already running.

Steps to reproduce actual behaviour: Try it with a script that runs for a long time or contains an infinite loop.

Video (might help if you don't know what I mean): https://www.youtube.com/watch?v=ZEkpM3VxFWU

Note: I haven't tried this with multiple sectors.
Note: I have included swimming before compiling, so, even though it is very unlikely, this might be an issue that only occurs with swimming included.

**Comment by maxteufel** [comment]
This mostly needs discussion, but I'm in favor of not stopping the script and not triggering it again, at least until we have a recommendation on long-running scripts. @tobbi, what do you think?

**Comment by tobbi** [comment]
I agree with you, @maxteufel. Restarting the script doesn't make sense, especially when the script is important for tracking level progress.
